### PR TITLE
Allow organization routes for commonauth and logincontext servlets

### PR DESCRIPTION
**Pull request by sadilchamishka**
### Proposed changes in this pull request

$Subject

### When should this PR be merged

Depends on this PR

 - https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/205

### CODE REVIEW

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 1]
```diff
@@ -2782,6 +2785,10 @@
                 <BasePath>/console/</BasePath>
             </Context>
         </WebApp>
+        <Servlet>
+            <Context>/commonauth(.*)</Context>
```
Comment (AnuradhaSK) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2789]: In addition to the added default servlets, let's template this part to add new servlets via deployment.toml

Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2795]: Fixed by - https://github.com/wso2/carbon-identity-framework/pull/4239/commits/6e938ab0ef0d21451a47955f58b1813b7bc8ce2b

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2 [hunk 2]
```diff
@@ -2782,6 +2785,16 @@
                 <BasePath>/console/</BasePath>
             </Context>
         </WebApp>
+        <Servlet>
+        {% for servlet in tenant_context.rewrite.servlets %}
```
Comment (AnuradhaSK) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2:2789]: Shouldn't tenant_context be organization_context? check other places as well
```suggestion
        {% for servlet in organization_context.rewrite.servlets %}
```

Comment (sadilchamishka) [reply features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2]: Yes. fixed by - https://github.com/wso2/carbon-identity-framework/pull/4239/commits/6b320902fb5deffe07c447086292a2645a72b2c9

**Review by AnuradhaSK — APPROVED** [review]

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**